beamline_service/src/main.py

```diff
@@ -22,95 +27,175 @@ def init_logging():
 MONGO_DB_USERNAME = str(os.environ.get('MONGO_INITDB_ROOT_USERNAME', default=""))
 MONGO_DB_PASSWORD = str(os.environ.get('MONGO_INITDB_ROOT_PASSWORD', default=""))
 MONGO_DB_URI = "mongodb://%s:%s@mongodb_bl531:27017/?authSource=admin" % (MONGO_DB_USERNAME, MONGO_DB_PASSWORD)
```

Comment:
Would it be better for storing the `MONGO_DB_URL` in environment variable?

Reply:
I updated this in this file, docker_compose.yml file and README.md